sorc/build_workflow_utils.sh

```diff
@@ -5,35 +5,19 @@ set -eux
 readonly UTILS_DIR=$(cd "$(dirname "$($cmd -f -n "${BASH_SOURCE[0]}" )" )" && pwd -P)
 
 # Adapt for global-workflow structure.
-target=${target:-"NULL"}
+source ${UTILS_DIR}/machine-setup.sh > /dev/null 2>&1
 modulefile=${UTILS_DIR}/../modulefiles/workflow_utils.$target
 if [[ -f $modulefile ]]; then
   set +x
-  source ${UTILS_DIR}/machine-setup.sh > /dev/null 2>&1
   source $modulefile
   module list
   set -x
+else
+  echo "FATAL: modulefile $modulefile not found!"
+  exit 1
```

Comment:
This assumes that on every machine there has to be a modulefile.
If the software stack is in path, a modulefile is not required.
Which machine is this failing on?

Reply:
The need for this came to me based on the above comment. In the context of global-workflow, I don't know that manually loading a module beforehand is a use case to support, and I'd prefer the error to specify the actual problem rather than cmake just failing.